src/scripts/main.js

```diff
@@ -1 +1,78 @@
-'use strict';
+function showNotification(message) {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.textContent = message;
+  document.body.appendChild(notification);
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  let resolved = false;
+  const clickHandler = (mouseEvent) => {
+    if (mouseEvent.button === 0) {
+      resolved = true;
+      resolve('First promise was resolved');
+      document.removeEventListener('click', clickHandler);
+    }
+  };
+
+  document.addEventListener('click', clickHandler);
+
+  setTimeout(() => {
+    if (!resolved) {
+      reject(new Error('First promise was rejected'));
+      document.removeEventListener('click', clickHandler);
+    }
+  }, 3000);
+});
+
+firstPromise
+  .then((message) => showNotification('success', message))
+  .catch((error) => showNotification('error', error));
```

Comment:
The `showNotification` function is called with two arguments ('success', message) and ('error', error), but the function is defined to accept only one parameter. This will lead to incorrect behavior. Consider updating the function definition to handle two parameters or modify these calls.